### PR TITLE
Gillespy2 v1.5.9 compatibility fixes

### PR DESCRIPTION
**Pull request by BryanRumsey**

### CODE REVIEW

**Review by ethangreen-dev — APPROVED** [review]
Looks good!

File: stochss/handlers/util/ensemble_simulation.py [hunk 1]
```diff
@@ -59,9 +59,12 @@ def __init__(self, path, preview=False):
 
 
     def __get_run_settings(self):
-        solver_map = {"SSA":SSACSolver(model=self.g_model), "Tau-Leaping":TauLeapingSolver,
+        solver_map = {"SSA":SSACSolver, "Tau-Leaping":TauLeapingSolver,
```
Comment (jtcooper10) [inline stochss/handlers/util/ensemble_simulation.py:62]: I think the new C++ solvers (`TauLeapingCSolver` and `ODECsolver`) are available in GillesPy2 v1.5.10, do we want to include those?

**Review by jtcooper10 — APPROVED** [review]
Overall looks good to me, just left one comment.

File: stochss/handlers/util/ensemble_simulation.py (hunk 1 above)
Comment (BryanRumsey) [reply stochss/handlers/util/ensemble_simulation.py:62]: That will be in the 2.3.1 release

Comment (jtcooper10) [reply stochss/handlers/util/ensemble_simulation.py:62]: Ah, right! I see the Issue now.

**Pull request merged**